src/fdosecrets/objects/Service.cpp

```diff
@@ -58,9 +58,11 @@ namespace FdoSecrets
     bool Service::initialize()
     {
         if (!QDBusConnection::sessionBus().registerService(QStringLiteral(DBUS_SERVICE_SECRET))) {
-            qDebug() << "Another secret service is running";
-            emit error(tr("Failed to register DBus service at %1: another secret service is running.")
-                           .arg(QLatin1Literal(DBUS_SERVICE_SECRET)));
+            auto otherService = m_plugin->runningServiceInfo();
+            emit error(tr("Failed to register DBus service at %1: another secret service is running.<br/>"
+                          "<i>%2</i><br/>"
+                          "Please stop/remove it before re-enabling the Secret Service Integration.")
```

Comment:
Start a new `tr(...)` here

Reply:
So I'm going to save the checking PID to 2.6.0 anyway. But I don't quite get it. Do you mean something like
```
tr("Failed to register DBus service at %1: another secret service is running.<br/>")
+ tr("<i>%2</i><br/>")
+ tr("Please stop/remove it before re-enabling the Secret Service Integration.")
```
What's the benefit over a single `tr(...)` call?

(I'm not objecting. I just want to learn about what's the caveat here)

Reply:
Remove the middle `tr(...)`. For this PR the benefit is the initial part of the message will still translate properly. We also want to avoid adding pure fill ins to translate.

Reply:
I see. Thanks for the explanation.